test-clients/src/main/java/com/hedera/services/bdd/suites/file/FileUpdateSuite.java

```diff
@@ -772,6 +777,142 @@ private HapiApiSpec messageSubmissionSizeChange() {
+                                .fee(ONE_HBAR)
+                                .signedBy(GENESIS)
+                                .hasPrecheck(INVALID_SIGNATURE),
+                        cryptoTransfer(tinyBarsFromTo(STILL_HOLLOW, FUNDING, 1))
+                                .payingWith(GENESIS)
+                                .fee(ONE_HBAR)
+                                .signedBy(GENESIS),
+                        // Invalid hollow creation as hbar transfer list repeats a new EVM address
+                        cryptoTransfer(
+                                        (spec, b) ->
+                                                b.getTransfersBuilder()
+                                                        .addAccountAmounts(aaWithNum(2, -1000))
+                                                        .addAccountAmounts(
+                                                                aaWith(bNewEvmAddress, +500))
+                                                        .addAccountAmounts(
+                                                                aaWith(bNewEvmAddress, +500)))
+                                .signedBy(GENESIS)
+                                .hasPrecheck(ACCOUNT_REPEATED_IN_ACCOUNT_AMOUNTS),
+                        // Invalid hollow creation as fungible transfer list repeats the new EVM
+                        // address
+                        cryptoTransfer(
+                                        (spec, b) -> {
+                                            b.getTransfersBuilder()
+                                                    .addAccountAmounts(aaWithNum(2, -1000))
+                                                    .addAccountAmounts(
+                                                            aaWith(bNewEvmAddress, -1000));
+                                            b.addTokenTransfers(
+                                                    TokenTransferList.newBuilder()
+                                                            .setToken(fungibleTokenId.get())
+                                                            .addTransfers(aaWithNum(2, -1000))
+                                                            .addTransfers(
+                                                                    aaWith(cNewEvmAddress, +500))
+                                                            .addTransfers(
+                                                                    aaWith(cNewEvmAddress, +500)));
+                                        })
+                                .signedBy(GENESIS)
+                                .hasPrecheck(INVALID_ACCOUNT_AMOUNTS),
+                        // Valid hollow creation with the new EVM address in both the hbar transfer
+                        // list *and* a token transfer list; fails with
+                        // TOKEN_NOT_ASSOCIATED_TO_ACCOUNT
+                        cryptoTransfer(
```

Comment:
Is this an outdated comment, it seems to be working now with the changes from the base branch?

Reply:
Yes, another outdated item. 😞